controllers/release/adapter.go

```diff
@@ -720,13 +720,16 @@ func (a *adapter) getCollectorsPipelineRunBuilder(pipelineType, namespace, revis
 
 // createManagedCollectorsPipelineRun creates a PipelineRun to run the collectors Pipeline for collectors in the ReleasePlanAdmission.
 func (a *adapter) createManagedCollectorsPipelineRun(releasePlanAdmission *v1alpha1.ReleasePlanAdmission) (*tektonv1.PipelineRun, error) {
+	url, err := releasePlanAdmission.Spec.Pipeline.PipelineRef.GetUrl()
+	if err != nil {
```

Comment:
This doesn't make sense. If there is *no* error, then we overwrite?

Reply:
to me, `if err != nil` means there is an error getting the URL ... therefore fallback on a default/hardcoded value.